client/src/sync-filesystem.js

```diff
@@ -162,11 +379,37 @@ function SyncFileSystem(fs) {
           return callback(err);
         }
 
-        if(conflicted) {
-          conflict.removeFileConflict(fs, newPath, callback);
-        } else {
-          callback();
+        if(!conflicted) {
+          return callback();
         }
+
+        conflict.removeFileConflict(fs, newPath, function(err) {
+          if(err) {
+            return callback(err);
+          }
+
+          fsUtils.getPathsToSync(fs, root, function(err, pathsToSync) {
+            var indexInPathsToSync;
+            var syncInfo;
+
+            if(err) {
+              return callback(err);
+            }
+
+            indexInPathsToSync = findPathIndexinArray(pathsToSync.toSync, newPath);
+
+            if(indexInPathsToSync === -1) {
+              return;
+            }
+
+            syncInfo = pathsToSync.toSync[indexInPathsToSync];
+            syncInfo.type = syncModes.CREATE;
```

Comment:
Why mix the names?  Sometimes you call this a `mode` and sometimes a `type`.  Why not stick with one?